CONTRIBUTING.md

```diff
@@ -135,23 +139,15 @@ master constitutes a tagged release.
       the changelog. To lookup an alias from an email, try `bash
       ./scripts/authors.sh <email>`
     - reset the `CHANGELOG_PENDING.md`
-- bump versions
-- push to release/vX.X.X to run the extended integration tests on the CI
-- merge to master
-- merge master back to develop
+    - bump versions
+- push latest develop with prepared release details to release/vX.X.X to run the extended integration tests on the CI
+- if necessary, make pull requests against release/vX.X.X and squash merge them
```

Comment:
see: https://github.com/tendermint/tendermint/pull/3503/files#r269982992
There are edge cases where we do want to merge instead of squash (we can either disallow these cases or use merge).

Reply:
this should say "new pull requests, based on the release branch, against the release branch".

The only case we don't want to squash merge (afaik) is when we need to merge develop to the release branch. But in general we should be able to avoid that by basing those fixes on the release branch in the first place, yeh ?

Reply:
>  in general we should be able to avoid that by basing those fixes on the release branch in the first place, yeh ?

Yes, makes sense. We should base PRs directly on the release branch if we are sure, we want to include them.